.github/actions/sam-build/action.yaml

```diff
@@ -0,0 +1,24 @@
+name: sam-build
+
+runs:
+  # This creates a composite action to be used as a step in a job
+  # https://docs.github.com/en/actions/creating-actions/creating-a-composite-action
+  using: "composite"
+  steps:
+    # Convert Pipfile.lock to requirements.txt for sam
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - run: pip install -U pipenv
+      shell: bash
+    - run: pipenv requirements > requirements.txt
+      shell: bash
```

Comment:
you could probably just use pipenv action here instead of two runs? https://github.com/marketplace/actions/pipenv-for-github-actions

Reply:
unfortunately we don't have a way of specifying the python version when using that action, and the other pipenv actions don't provide a way of generating `requirements.txt`.